apis/v1alpha1/opampbridge_types.go

```diff
@@ -105,6 +105,9 @@ type OpAMPBridgeSpec struct {
 	// https://kubernetes.io/docs/concepts/workloads/pods/pod-topology-spread-constraints/
 	// +optional
 	TopologySpreadConstraints []v1.TopologySpreadConstraint `json:"topologySpreadConstraints,omitempty"`
+	//PodDNSConfig defines the DNS parameters of a pod in addition to those generated from DNSPolicy.
+	//https://github.com/kubernetes-client/python/blob/master/kubernetes/docs/V1PodDNSConfig.md
```

Comment:
Do we need this?

Reply:
Originally the issue doesn't request adding PodDNSConfig for TA nor OpAMPBridge, but I've tried to proactively enable it in case of future issues.

Reply:
I meant the link to the other GitHub repo

Reply:
It was only a help link, but if you don't see it as valuable information, I can remove it.

Reply:
Why do we link the python docs specifically?

Reply:
It's not related to Python precisely. I've just tried putting a help link to the DNSPolicy docs.

Reply:
I removed the link.